src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import {
+  Layout,
+  ProtectedRoute,
+  SplashScreen,
+  Login,
+  Register,
+  Doctors,
+  DoctorDetails,
+  Appointments,
+  NewAppointment,
+} from './components';
+
+function App() {
```

Comment:
Hello @MMhaidari , talking about this point, I'm afraid I wasn't able to check this important point in your project, this is because when I tried to enter a new user on the sign up page, this didn't work, anyway I'll create a new issue in my PR and this point will be pending for the next reviewer after me

Reply:
My bad, now I was able to enter with the seeds credentials:

Jiya
Jiya@gmail.com
Jiya123

I consider this could be added to the description for the next code review, anyway, I was able to solve this problem and continue for the CR, it looks like this issue was solved! thanks for that! 😄